models/userroom.js

```diff
@@ -1,31 +1,35 @@
 Chatter.UserRoom = ChatterUserRoom = Astro.Class({
-    name: "ChatterUserRoom",
-    collection: new Mongo.Collection("chatteruserroom"),
-
-    fields: {
-        userId: {
-            type: "string"
-        },
-
-        roomId: {
-            type: "string"
-        },
-
-        count: {
-            type: "number",
-            default: function() {
-              return 0;
-            }
-        }
-    },
+  name: "ChatterUserRoom",
+  collection: new Mongo.Collection("chatteruserroom"),
 
-    events: {
+  fields: {
+    userId: {
+      type: "string"
+    },
 
+    roomId: {
+      type: "string"
     },
 
-    methods: {
+    count: {
```

Comment:
What are you counting here? Wouldn't this be a 1-1 relation that either exists or not?